viewing_party/party.py

```diff
@@ -1,23 +1,219 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    my_movie = {
+        "title" : movie_title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+    for keys in my_movie:
+        if my_movie[keys] == False:
+            return None
+    
+    return my_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+
+    return user_data    
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    title = user_data["watchlist"][0]

```

Comment:
I believe this returns a movie not a movie title. There's a bit of confusion I think. This function reassigns the `title` parameter value as the first item of the watchlist (which should be a movie, if it exists). Then that item is added to the `watched` list and the `watchlist` is set to an empty list. I think what you want to do is iterate over the `watchlist` items (i.e. movies) and if the title of that item is the same as the param `title` then append it to the `watched` list and remove it from the `watchlist`. Note that if you do that, it's a good idea to `break` after those modifications because changing the list you're iterating over can have unintended consequences.